.github/workflows/pull-request.yml

```diff
@@ -22,8 +26,7 @@ jobs:
           node-version: '16.x'
           registry-url: 'https://registry.npmjs.org'
       - run: npm install
-      - run: chromium --version
       - run: npm run build-test
       - name: Perform CodeQL Analysis
+        if: matrix.os == 'ubuntu-latest' # we only need to run this check on one platform
```

Comment:
nit: any indentation needed below this line?

Reply:
nit: do we need a hyphen before `if`?

Reply:
> How do we test the GitHub Action?

You can actually see it running in this PR - if you scroll down, you'll see the tests being run on the three platforms specified. 

I can update the PR description if it's not very clear as to how I did it, but I used my forked repository to test after enabling github actions there.

Reply:
A hyphen indicates a command or a step to execute, so we don't want a hyphen on `if` as we want to conditionally run the command it applies to.

Also, the indentation looks fine to me, but let me know if I'm missing something

Reply:
we don't need `use` to be a part of the `if` block?

Reply:
The `if` applies to the whole block, which is scoped by the hyphen